guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -482,6 +482,10 @@ $ bundle exec rake TEST=test/cases/associations/has_many_associations_test.rb
 
 You can invoke `test_jdbcmysql`, `test_jdbcsqlite3` or `test_jdbcpostgresql` also. See the file `activerecord/RUNNING_UNIT_TESTS.rdoc` for information on running more targeted database tests.
 
+#### Using Debuggers with Test
+
+To use an external debugger(pry, byebug, etc), install the debugger and use the debugger as normal.  If  debugger performance issues occur, run tests in serial by setting `PARALLEL_WORKERS=1` or run a single test with `-n test_long_test_name`.
```

Comment:
Thanks for opening a PR! I think this reads a little more clearly. I removed "performance" because it's not quite a performance issue when using a debugger with parallel testing. They really aren't designed to work together. 

When you make the changes, please squash your commits into 1. Thank you!

```suggestion
To use an external debugger (pry, byebug, etc), install the debugger and use it as normal.  If debugger issues occur, run tests in serial by setting `PARALLEL_WORKERS=1` or run a single test with `-n test_long_test_name `.
```

Reply:
Thank you for the suggestion!  I will get these commits sqaushed by EoD :)

Reply:
Done!  Thanks :)